test/integration/ClientNotificationMongoRepoSpec.scala

```diff
@@ -181,7 +181,7 @@ class ClientNotificationMongoRepoSpec extends UnitSpec
       await(repository.save(client1Notification2))
       collectionSize shouldBe 2
 
```

Comment:
not sure of the contect of `client1Notification1.copy(_id = Some(BSONObjectID.generate())` WRT using CN for delete